examples/synthetic_benchmark.cpp

```diff
@@ -357,7 +357,7 @@ void synthetic_benchmark_main(int nranks, int rank)
             }
         }
 
-        x += do_work(i);
+        x = x + do_work(i);
```

Comment:
This ended up being preferable to returning x?

Reply:
Returning x is likely preferable.  This was a temporary work-around to a non-Levelzero issue that was erroneously added.  The full fix should be in a separate PR.